app/assets/javascripts/application.js

```diff
@@ -13,3 +13,11 @@
 //= require jquery
 //= require jquery_ujs
 //= require_tree .
+
+const foo = 2;
+let baz = 1;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).